spec/index.bs

```diff
@@ -16732,6 +16733,23 @@ expose real values whenever the feature is available on the adapter:
 - New WGSL extensions:
     - [=extension/subgroups=]
 
+<h3 id=read-write-storage-texture-format-tier2 data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"read-write-storage-texture-format-tier2"`
+</h3>
+
+Allows the {{GPUStorageTextureAccess/"read-write"}} {{GPUStorageTextureAccess}} on the textures with
+one of the below {{GPUTextureFormat}}s:
+- {{GPUTextureFormat/"rgba32float"}}
+- {{GPUTextureFormat/"rgba32uint"}}
+- {{GPUTextureFormat/"rgba32sint"}}
+- {{GPUTextureFormat/"rgba16float"}}
+- {{GPUTextureFormat/"rgba16uint"}}
+- {{GPUTextureFormat/"rgba16sint"}}
+- {{GPUTextureFormat/"rgba8unorm"}}
+- {{GPUTextureFormat/"rgba8uint"}}
+- {{GPUTextureFormat/"rgba8sint"}}
```

Comment:
nit: List these in the same order they're in the table

I've verified the table matches this list at this revision.